### DISCUSSION TITLE
Ordre des champs dans le fichier de JSON

### DISCUSSION THREAD

**Discussion by ChrisThe01** [MTESSDev/FRW#144] [Annonces]
Bonjour,

Nous avons remarqué que l'ordre les champs dans le fichier JSON de sortie n'est pas la même que ceux présenté dans le formulaire. Nous soupçonnons que les champs sont présent dans le JSON en ordre de saisie et non en ordre de présentation. Ceci nous cause des problèmes. Il faudra que les champs dans le JSON soient dans le même que présenté dans le formulaire peut importe l'ordre de saisie. Comment doit-on faire ?

Merci,

**Comment by anisite** [comment]
Bonsoir,

Effectivement l'ordre de saisie dicte celle de sortie. Normalement pour les traitements cela n'a pas d'impact, quel est le problème avec le principe actuel au juste pour que cela vous cause des soucis?

Merci

**Reply by ChrisThe01** [reply]
Effectivement ça nous causait des problèmes. Je crois, en parlant avec Marc, que nous avons une solution de rechange.

Merci,

**Comment by anisite** [comment]
Bonjour,

Pour garder une trace, une des solutions proposée est d'utiliser l'ordre des questions extraites puisque celle-ci correspond à celle du formulaire.

Merci
